drivers/wifi/esp_at/esp.c

```diff
@@ -866,11 +866,11 @@ MODEM_CMD_DIRECT_DEFINE(on_cmd_ipd)
 {
 	struct esp_data *dev = CONTAINER_OF(data, struct esp_data,
 					    cmd_handler_data);
-	struct esp_socket *sock;
-	int data_offset;
-	long data_len;
-	int err;
-	int ret;
+	struct esp_socket *sock = NULL;
+	int data_offset = 0;
+	long data_len = 0;
+	int err = 0;
```

Comment:
No need to initialize the err variable as value to it is set few lines below.

Reply:
👌